core/src/avm2/globals/flash/net/NetStreamPlayTransitions.as

```diff
@@ -0,0 +1,13 @@
+package flash.net
+{
+	public final class NetStreamPlayTransitions
```

Comment:
The class is version-gated to FP10, and based on the documentation, `APPEND_AND_WAIT` and `RESUME` are both gated to FP10.1. Can you apply the versioning like https://github.com/ruffle-rs/ruffle/blob/master/core/src/avm2/globals/README.md#api-versioning specifies? The class should have the metadata `[API("662")]`, and `APPEND_AND_WAIT` and `RESUME` should have `[API("667")]`. Also, looking at the documentation, it doesn't seem like the class is `final`.

Reply:
Thanks for your comment, I'd forgotten about the versions!